lib/excoveralls/cobertura.ex

```diff
@@ -0,0 +1,213 @@
+defmodule ExCoveralls.Cobertura do
+  @moduledoc """
+  Generate XML Cobertura output for results.
+  """
+
+  alias ExCoveralls.Settings
+  alias ExCoveralls.Stats
+
+  @file_name "cobertura.xml"
+
+  @doc """
+  Provides an entry point for the module.
+  """
+  def execute(stats, options \\ []) do
+    stats
+    |> generate_xml(Enum.into(options, %{}))
+    |> write_file(options[:output_dir])
+
+    ExCoveralls.Local.print_summary(stats)
+
+    Stats.ensure_minimum_coverage(stats)
+  end
+
+  defp generate_xml(stats, _options) do
+    prolog = [
+      "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n",
+      "<!DOCTYPE coverage SYSTEM \"http://cobertura.sourceforge.net/xml/coverage-04.dtd\">\n"
+    ]
+
+    timestamp = DateTime.utc_now() |> DateTime.to_unix(:millisecond)
+
+    version = "1.9.4.1"
```

Comment:
Could you help me understand what this version mean? (description or reference document)

Reply:
Also, if this version is meaningful for end users, it's worthwhile to note in the README?

Reply:
Cobertura is the name of the java software that generates those coverage reports. The version on the XML file is the version of the tool. Since we are not using the tool, that value is not meaningful but in the [DTD schema](https://cobertura.sourceforge.net/xml/coverage-04.dtd) is marked as required so I put there the last version of cobertura I'm aware of.

The same is done in the erlang [covertool](https://github.com/covertool/covertool) package for instance.

Reply:
Thanks for the comment. 

> Since we are not using the tool, that value is not meaningful but in the [DTD schema](https://cobertura.sourceforge.net/xml/coverage-04.dtd) is marked as required so I put there the last version of cobertura I'm aware of.

I am looking at the following page, but it shows some description around version 2 (ex. `2.1.1`), dated around 2015. Does it make sense to put `2.1.1`? or put in a less-meaningful values rather than putting the last version of `1.9.x`?

- https://github.com/cobertura/cobertura
- https://github.com/cobertura/cobertura/tags

I understood that this version itself isn't meaningful, but just wanted to clarify the current meaning, as I am not so familiar with the current status of cobetura (though I once used the tool some time ago).

Reply:
Yes, I'm not so familiar with this tool too. It makes sense to put the latest version. I would also put something like `0.0.0` to document that we are not using that field but then I don't know if we break some possible usage.

For the moment I'll change it to 2.1.1